cns/restserver/ipam.go

```diff
@@ -190,6 +190,49 @@ func (service *HTTPRestService) GetPodIPConfigState() map[string]cns.IPConfigura
 	return service.PodIPConfigState
 }
 
+func (service *HTTPRestService) GetHTTPRestDataHandler(w http.ResponseWriter, r *http.Request) {
+	var (
+		req           string
+		resp          GetHTTPServiceDataResponse
+		statusCode    int
+		returnMessage string
+		err           error
+	)
+
+	statusCode = UnexpectedError
```

Comment:
this is only used when there is an error from the response. no need to keep an extra variable, just 
```go
if err != nil {
    resp.Response.ReturnCode = UnexpectedError
    ....
}
```

Reply:
ack, addressed this